### PR TITLE
Scenario risk

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1389265. Here I am simply updating the calculator according to the latest API changes in hazardlib and risklib and adding tests.
The tests are running here: https://ci.openquake.org/job/zdevel_oq-risklib/373

### CODE REVIEW

File: openquake/commonlib/calculators/calc.py [hunk 1]
```diff
@@ -314,6 +315,7 @@ def expand(data_dict, sites=None):
             for key, data_by_imt in data_dict.iteritems()}
 
 
+# not used anymore
```
Comment (ptormene) [inline openquake/commonlib/calculators/calc.py]: Should we remove it if it's not used anymore?

File: openquake/commonlib/readinput.py [hunk 2]
```diff
@@ -656,6 +660,8 @@ def get_exposure(oqparam):
         set(['occupants'])
     asset_refs = set()
     time_event = getattr(oqparam, 'time_event', None)
+    ignore_missing_costs = set(getattr(oqparam, 'ignore_missing_costs', []))
```
Comment (ptormene) [inline openquake/commonlib/readinput.py]: Even if the name is already long, you might consider renaming `ignore_missing_costs` -> `ignore_missing_cost_types`

**Comment by ptormene** [comment]
Apart from the very minor comment, LGTM

File: openquake/commonlib/calculators/calc.py (hunk 1 above)
Comment (micheles) [inline openquake/commonlib/calculators/calc.py]: Removed

**Pull request merged**